lib/rex/post/meterpreter/ui/console/command_dispatcher/mimikatz.rb

```diff
@@ -159,7 +159,7 @@ def get_privs
   end
 
   def system_check
-    unless (client.sys.config.getuid == "NT AUTHORITY\\SYSTEM")
+    unless (client.sys.config.is_system?)
```

Comment:
Minor nitpick: parens not required here :)

Reply:
removed :)